wlc/__init__.py

```diff
@@ -443,6 +443,10 @@ def changes(self):
     def delete(self):
         self.weblate.raw_request("delete", self._url)
 
+    def add_source_string(self, component, msgid, msgstr):
+        """Adds a source string to a monolingual base file"""
+        return self.weblate.add_source_string(self.slug, component, msgid, msgstr)
+
```

Comment:
This confuses me a bit. Why add this here and not directly on the component?

Reply:
Good catch! Have implemented as suggested.